plugins/copr.py

```diff
@@ -463,21 +464,55 @@ def _download_repo(self, project_name, repo_filename, chroot=None):
 
         for line in f:
             if re.match(r"\[copr:", line):
-                repo_filename = os.path.join(self.base.conf.get_reposdir,
-                                             "_" + line[1:-2] + ".repo")
+                self.repo_filename = os.path.join(self.base.conf.get_reposdir,
+                                                  "_" + line[1:-2] + ".repo")
             break
 
         # if using default hub, remove possible old repofile
         if self.copr_url == self.default_url:
             # copr:hub:user:project.repo => _copr_user_project.repo
-            old_repo_filename = repo_filename.replace("_copr:", "_copr", 1)\
+            old_repo_filename = self.repo_filename.replace("_copr:", "_copr", 1)\
                 .replace(self.copr_hostname, "").replace(":", "_", 1).replace(":", "-")\
                 .replace("group_", "@")
             if os.path.exists(old_repo_filename):
                 os.remove(old_repo_filename)
 
-        shutil.copy2(f.name, repo_filename)
-        os.chmod(repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+        shutil.copy2(f.name, self.repo_filename)
+        os.chmod(self.repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
+
+    def _runtime_deps_warning(self, project_name, chroot=None):
+        repo_file = ConfigParser(strict=False)
+        repo_file.read(self.repo_filename)
+
+        runtime_deps = []
+        for repo in repo_file.sections():
+            # repository is the project itself
+            if repo in self.repo_filename:
+                continue
```

Comment:
This can not work for repositories that contain `:ml`, right?  Also, old
copr versions had different variant of repoid/filenames.

Cam you invent something like repoid starting with  `coprdep:....` on frontend,
and do the if based on that?

Reply:
You're right, repositories with `:ml` wouldn't work. Changed the repo ID as you suggested.